qdrant-landing/content/documentation/fastembed/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "FastEmbed" 
+weight: 6
+---
+
+# What is FastEmbed?
+FastEmbed is a lightweight Python library built for embedding generation. It supports popular embedding models and offers a user-friendly experience for embedding data into vector space. 
+
+By using FastEmbed, you can ensure that your embedding generation process is not only fast and efficient but also highly accurate, meeting the needs of various machine learning and natural language processing applications.
+
+FastEmbed easily integrates with Qdrant for a variety of multimodal search purposes.
+
+## How to get started with FastEmbed
+
+|Beginner|Advanced|
+|:-:|:-:|
+|[Generate Text Embedings with FastEmbed](fastembed-quickstart/)|[Combine FastEmbed with Qdrant for Vector Search](fastembed-semantic-search/)|
+
+## Why is FastEmbed popular?
+
+- Light & Fast: FastEmbed is designed to be efficient and quick, utilizing quantized model weights to reduce the computational load.
```

Comment:
quantized model weights -> we have some quantized models, but they are usually not quantized

regarding loght, we might want to highlight that it does not have that many dependencies and can be considered being a light-weight package (comparing to the other inference frameworks which require torch / tf / etc.)